packages/cli/test/integration/generators/negative-tests.relation.integration.js

```diff
@@ -0,0 +1,131 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const path = require('path');
+const assert = require('yeoman-assert');
+const testlab = require('@loopback/testlab');
+const fs = require('fs');
+const expect = testlab.expect;
+const TestSandbox = testlab.TestSandbox;
+const generator = path.join(__dirname, '../../../generators/relation');
+const SANDBOX_FILES = require('../../fixtures/relation').SANDBOX_FILES;
+const testUtils = require('../../test-utils');
+
+// Test Sandbox
+const SANDBOX_PATH = path.resolve(__dirname, '..', '.sandbox');
+const MODEL_APP_PATH = 'src/models';
+const CONTROLLER_PATH = 'src/controllers';
+const REPOSITORY_APP_PATH = 'src/repositories';
+const sandbox = new TestSandbox(SANDBOX_PATH);
+
+describe('lb4 relation negative tests', function() {
```

Comment:
We don't split the tests to positive and negative. IIUC, these tests are not specific to any relation type but are the same for all relation types, this they should go to file called `packages/cli/test/integration/generators/relation.integration.js`. I thought there already is one, but as I see, there are multiple `*.relation.integration.js` test files instead - which is great! So let's rename this new file `negative-tests.relation.integration.js` to `relation.integration.js`.

Let's also rename the test block accordingly.

```ts
describe('lb4 relation', function() {
```

Reply:
done